src/services/railgun/core/load-provider.ts

```diff
@@ -41,32 +37,20 @@ const createFallbackProviderForNetwork = async (
   return fallbackProvider;
 };
 
-const createPollingProviderForNetwork = async (
-  networkName: NetworkName,
-  fallbackProvider: FallbackProvider,
-  pollingInterval: number,
-): Promise<PollingJsonRpcProvider> => {
-  const existingProvider = pollingProviderMap[networkName];
-  if (existingProvider) {
-    return existingProvider;
-  }
-  const network = NETWORK_CONFIG[networkName];
-  if (!isDefined(network)) {
-    throw new Error('No network found');
-  }
-  const pollingProvider = await createPollingJsonRpcProviderForListeners(
-    fallbackProvider,
-    network.chain.id,
-    pollingInterval,
-  );
-  setPollingProviderForNetwork(networkName, pollingProvider);
-  return pollingProvider;
-};
-
+/**
+ * 
+ * @param chain 
+ * @param networkName 
+ * @param fallbackProviderJsonConfig 
+ * @param pollingInterval - DEPRECATED
+ */
 const loadProviderForNetwork = async (
   chain: Chain,
   networkName: NetworkName,
   fallbackProviderJsonConfig: FallbackProviderJsonConfig,
+  /**
+   * @deprecated pollingInterval - DEPRECATED
+   */
   pollingInterval: number,
```

Comment:
Add deprecated to this too?

Reply:
It's already there if you are asking to ensure it has deprecated on it